src/password.php

```diff
@@ -4,23 +4,23 @@
 ?>
 <link rel="stylesheet" type="text/css" href="css/dataTables.bootstrap.min.css">
 <link rel="stylesheet" type="text/css" href="css/responsive.dataTables.min.css">
-<script type="text/javascript" src="js/setlocalstorage.js"></script>
-<script type="text/javascript" src="js/lib/jquery.dataTables.min.js"></script>
-<script type="text/javascript" src="js/lib/dataTables.bootstrap.min.js"></script>
-<script type="text/javascript" src="js/lib/dataTables.responsive.min.js"></script>
-<script type="text/javascript" src="js/lib/FileSaver.min.js"></script>
-<script type="text/javascript" src="js/crypto/aes.js"></script>
-<script type="text/javascript" src="js/crypto/sha512.js"></script>
-<script type="text/javascript" src="js/crypto/pbkdf2.js"></script>
-<script type="text/javascript" src="js/crypto/password.js"></script>
-<script type="text/javascript" src="js/lib/jquery.csv.js"></script>
-<script type="text/javascript" src="js/main.js"></script>
-<script type="text/javascript" src="js/plugin.js"></script>
-<script type="text/javascript" src="js/plugins/linkButton.js"></script>
-<script type="text/javascript" src="js/plugins/tags.js"></script>
-<script type="text/javascript" src="js/plugins/showPasswordAge.js"></script>
-<script type="text/javascript" src="js/plugins/keyboardShortcuts.js"></script>
-   <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+<script src="js/setlocalstorage.js"></script>
+<script src="js/lib/jquery.dataTables.min.js"></script>
+<script src="js/lib/dataTables.bootstrap.min.js"></script>
+<script src="js/lib/dataTables.responsive.min.js"></script>
+<script src="js/lib/FileSaver.min.js"></script>
+<script src="js/crypto/aes.js"></script>
+<script src="js/crypto/sha512.js"></script>
+<script src="js/crypto/pbkdf2.js"></script>
+<script src="js/crypto/password.js"></script>
+<script src="js/lib/jquery.csv.js"></script>
+<script src="js/main.js"></script>
+<script src="js/plugin.js"></script>
+<script src="js/plugins/linkButton.js"></script>
+<script src="js/plugins/tags.js"></script>
+<script src="js/plugins/showPasswordAge.js"></script>
+<script src="js/plugins/keyboardShortcuts.js"></script>
+   <nav class="navbar navbar-default navbar-fixed-top">
```

Comment:
why was `role="navigation"` removed. This is a standard needed for example for tools helping the blind.

Reply:
See this: https://stackoverflow.com/a/37215942/9253338

Reply:
Alright!